packages/components/src/components/multi-select/multi-select/multi-select-styles.ts

```diff
@@ -0,0 +1,194 @@
+import { Styles } from 'jss';
+import { getThemedFormStateColors } from '../../../styles/form-state-color-styles';
+import { FormState } from '../../../utils/form/form-state';
+import { getLabelStyles, INPUT_HEIGHT } from '../../../styles/form-styles';
+import { SelectDropdownDirectionInternal } from '../../../utils/select/select-dropdown';
+import { getPlaceholderStyles } from '../../../styles/placeholder';
+import { getNoResultsOptionJSSStyles, MULTI_SELECT_OPTION_HEIGHT } from '../../../styles/select/option-styles';
+import { getFunctionalComponentRequiredStyles } from '../../common/required/required-styles';
+import { getFunctionalComponentStateMessageStyles } from '../../common/state-message/state-message-styles';
+
+const inputYPadding = '13px';
+const selectorNativeSelect = '::slotted([slot=select])';
+
+export const getComponentCss = (
+  hasSelection: boolean,
+  direction: SelectDropdownDirectionInternal,
+  isOpen: boolean,
+  isDisabled: boolean,
+  hideLabel: BreakpointCustomizable<boolean>,
+  state: FormState,
+  isWithinForm: boolean,
+  hasLabel: boolean,
+  theme: Theme
+): string => {
+  const { primaryColor, contrastMediumColor, contrastHighColor, backgroundColor } = getThemedColors(theme);
+  const { formStateColor, formStateHoverColor } = getThemedFormStateColors(theme, state);
+  const isDirectionDown = direction === 'down';
+
+  return getCss({
+    '@global': addImportantToEachRule({
+      ':host': {
+        display: 'block',
+        position: 'relative',
+        ...hostHiddenStyles,
+      },
+      ...(isWithinForm && {
+        [selectorNativeSelect]: {
+          opacity: 0,
+          display: 'block',
+          height: '0px',
+        },
+      }),
+    }),
+    root: {
+      position: 'relative',
+    },
+    ...getListStyles(isOpen, direction, theme),
+    'input-container': {
+      display: 'flex',
+      background: backgroundColor,
+      transition: ['color', 'border-color', 'background-color'].map(getTransition).join(), // for smooth transitions between e.g. disabled states
+      cursor: isDisabled ? 'not-allowed' : 'text',
+      ...hoverMediaQuery({
+        '&:hover:not(.disabled)': {
+          borderColor: isOpen ? primaryColor : formStateHoverColor || primaryColor,
+        },
+      }),
+      '&:focus-within': {
+        borderColor: primaryColor,
+      },
+      border: `${borderWidthBase} solid ${isOpen ? primaryColor : formStateColor || contrastMediumColor}`,
+      borderRadius: borderRadiusSmall,
+      ...(isOpen && {
+        [isDirectionDown ? 'paddingBottom' : 'paddingTop']: '1px', // Add padding to keep same height when border changes
+        [isDirectionDown ? 'borderBottom' : 'borderTop']: addImportantToRule(`1px solid ${contrastMediumColor}`),
+        [isDirectionDown ? 'borderBottomLeftRadius' : 'borderTopLeftRadius']: 0,
+        [isDirectionDown ? 'borderBottomRightRadius' : 'borderTopRightRadius']: 0,
+      }),
+      ...getInputStyles(isDisabled, theme),
```

Comment:
I think this contains wrapping `@global` object. Shouldn't this be removed and then spreaded into local `@global` above?